ui/app/src/modules/Content/History/HistoryDialog.tsx

```diff
@@ -87,6 +88,19 @@ const translations = defineMessages({
   backToHistoryList: {
     id: 'historyDialog.back.selectRevision',
     defaultMessage: 'Back to history list'
+  },
+  confirmRevertTitle: {
+    id: 'historyDialog.confirmRevertTitle',
+    defaultMessage: 'Revert confirmation'
+  },
+  //TODO: discuss translations
+  confirmRevertBody: {
+    id: 'historyDialog.confirmRevertBody',
+    defaultMessage: 'Are you sure you want to revert to this version?'
+  },
+  confirmRevertPreviousBody: {
+    id: 'historyDialog.confirmRevertBody',
+    defaultMessage: 'Are you sure you want to revert to previous version?'
```

Comment:
What's up? 
Would be nice to include the title of the version to which the content would be reverted to